### PR TITLE
fix:change keysCommand default format auto

### PR DESCRIPTION
**Pull request by Elbehery**
This PR changes keysCommand default format from ascii-encoded to auto

resolves #https://github.com/etcd-io/bbolt/issues/588

cc @ahrtr

### CODE REVIEW

**Comment by Elbehery** [comment]
@ahrtr I updated this accordingly, ptal 👍🏽

**Comment by Elbehery** [comment]
@fuweid would appreciate your review :)

**Review by fuweid — APPROVED** [review]
LGTM

File: cmd/bbolt/main_test.go [hunk 1]
```diff
@@ -677,3 +709,11 @@ func convertInt64IntoBytes(num int64) []byte {
 	n := binary.PutVarint(buf, num)
 	return buf[:n]
 }
+
+func convertInt64IntoBytesString(nums ...int64) string {
```
Comment (ahrtr) [inline cmd/bbolt/main_test.go:713]:
```suggestion
func convertInt64KeysIntoHexString(nums ...int64) string {
```

File: cmd/bbolt/main_test.go [hunk 2]
```diff
@@ -319,37 +320,68 @@ func TestBucketsCommand_Run(t *testing.T) {
...
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			t.Logf("creating test database for subtest '%s'", tc.name)
+			db := btesting.MustCreateDB(t)
+
+			err := db.Update(func(tx *bolt.Tx) error {
+				t.Logf("creating test bucket %s", tc.testBucket)
+				b, bErr := tx.CreateBucketIfNotExists([]byte(tc.testBucket))
+				if bErr != nil {
+					return fmt.Errorf("error creating test bucket '%s': '%v'", tc.testBucket, bErr)
```
Comment (ahrtr) [inline cmd/bbolt/main_test.go:352]:
```suggestion
					return fmt.Errorf("error creating test bucket %q: %v", tc.testBucket, bErr)
```

**Comment by ahrtr** [comment]
Please also rebase this PR.

**Comment by Elbehery** [comment]
@ahrtr done 👍🏽

**Review by ahrtr — APPROVED** [review]
LGTM

**Comment by ahrtr** [comment]
Please update the readme in a followup PR.

**Comment by Elbehery** [comment]
> resolves ##588

@ahrtr  raised https://github.com/etcd-io/bbolt/pull/604

**Pull request merged**